composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "expectedbehavior/php-docraptor",
+    "description": "Official PHP wrapper for the DocRaptor API.",
+    "keywords": ["princexml", "php", "pdf", "xhtml", "xls", "xlsx", "docraptor"],
+    "homepage": "https://github.com/expectedbehavior/php-docraptor",
+    "type": "library",
+    "version": "dev",
```

Comment:
After testing this, we can release a version. It needs to be correct here, as well as in the composer.json and the installation instructions.